ting-bill-split_test.go

```diff
@@ -90,3 +90,37 @@ func TestParseMegabytes(t *testing.T) {
 		}
 	}
 }
+
+func TestParseBill(t *testing.T) {
+	cases := []struct {
+		in   string
+		want bill
+	}{
+		{
+			`minutes = 35.00
+messages = 8.00
+megabytes = 20.00
+devices = 42.00
+extras = 1.00
+fees = 12.84`,
+			bill{
+				Minutes:   35.00,
+				Messages:  8.00,
+				Megabytes: 20.00,
+				Devices:   42.00,
+				Extras:    1.00,
+				Fees:      12.84,
+			},
+		},
+	}
+
+	for _, c := range cases {
+		got, err := parseBill(strings.NewReader(c.in))
+		if !reflect.DeepEqual(got, c.want) {
+			t.Errorf("parseBill(%v) == %v, want %v", c.in, got, c.want)
+		}
+		if err != nil {
```

Comment:
The error check should be before the DeepEqual. If there's an error there's no point in checking the other result